src/main/java/app/controllers/OrderController.java

```diff
@@ -1,13 +1,36 @@
 package app.controllers;
 
+import app.entities.Customer;
 import app.persistence.ConnectionPool;
+import app.services.CarportSvg;
 import io.javalin.Javalin;
 import io.javalin.http.Context;
+import java.util.Locale;
 
 public class OrderController {
 
     public void addRoutes(Javalin app, ConnectionPool connectionPool) {
+        app.post("continuerequest", ctx -> continueRequest(ctx, connectionPool));
+    }
+
+    private void continueRequest(Context ctx, ConnectionPool connectionPool) {
+
+        Customer customer = ctx.sessionAttribute("currentUser");
```

Comment:
The AccountController is setup with `currentAccount` as it seemed to make more sense. You're welcome to comment if you disagree with this change